src/WooCommerceCheckout.php

```diff
@@ -118,4 +122,34 @@ public static function checkConfirmationEmailField() {
     }
   }
 
+  /**
+   * Check if we are using Amazon V2 in the checkout.
+   *
+   * @return bool
+   */
+  public static function isAmazonPayV2Checkout(): bool {
+    $is_amzon_pay_active = is_plugin_active('woocommerce-gateway-amazon-payments-advanced/woocommerce-gateway-amazon-payments-advanced.php') ?: FALSE;
+    if ($is_amzon_pay_active) {
+      if(version_compare(WC_AMAZON_PAY_VERSION, '2.0', '>=') && WC()->session->get('amazon_checkout_session_id') !== null) {
```

Comment:
This causes a fatal error on every cron run; see https://app.tideways.io/o/netzstrategen/gaco-stage/issues/errors?env=production&s=web&error=11305-902b0d55fddef6f8d651fe1035b7d4bd

![image](https://user-images.githubusercontent.com/41992/145245295-1b48a830-77b6-4da5-ab12-e10965b3ddec.png)

Reply:
`WC()->session` is being shown as undefined, probably because there is not a session in cron.

I should verify first it with  `isset(WC()->session)` or even `!WC()->session->has_session()`

What you think @sun ?